elements/pfe-cta/demo/demo.css

```diff
@@ -33,6 +33,9 @@ pfe-band[pfe-color="lightest"] pfe-card[pfe-color="lightest"] {
   margin-top: -10px;
   margin-left: -20px;
 }
+.padding {
```

Comment:
This isn't necessary - if you see my comment, it just needs the margin-left: 20px; added back to the selectors on line 40.

Reply:
I'd prefer not to add margins directly to the pfe-cta components because it can be misleading as to what the native behavior and styles actually are, and makes it difficult to tell what padding is coming from the container or the component. Since our traditional components (like a CTA) should always touch the edges of the parent container, I wanted to keep with that rule.

Reply:
I understand the concern but none of the components add margin to the host element as a rule.  I approached the layout of the page just like we do on the site and one dynamic layout approach we use is to apply negative margin to the parent and compensate for that on the components.  This shows that the CTA can play nicely in a standard layout like that.  I think it's very low risk that we'd be debugging a margin-issue.  If you feel strongly, I understand, could we remove the negative margins from the containers entirely then instead of adding padding classes?

Reply:
I think the consistent approach makes a lot of sense. Let the container control the layout.

Reply:
ohh, yes, sorry i did not see the negative margins on the card-layout. will do

Reply:
updated